src/test/resources/data/AutomatedSessionRemindersTest.json

```diff
@@ -22,7 +22,7 @@
       "googleId": "AutSessRem.instructor",
       "courseId": "AutSessRem.course",
       "name": "Test Ins for Aut Sessions Reminder",
-      "email": "${support.email}",
+      "email": "${test.instructor}@gmail.com",
```

Comment:
Pros of this approach:
Simplifies the tests a bit. That's always a good thing.
Speeds up the tests. Any idea by how much? I'm guessing a few seconds in total, but I could be wrong.

Cons of this approach:
1. Right now I don't have to check any additional email boxes because in the live server support.email is set up to reach my own inbox. If you change it like this, I have to check additional email boxes. That's what I'm trying to avoid. 
2. The original approach automatically confirms that the support.email is correctly directed to my email. There is a redirection involved in the path (from teammates@comp.... account to my gmail account) so a periodic validation of the path is useful because the path can break without me realizing it. e.g. due to spam filter added by SoC, expiry of unix account etc.
3. If we reuse an account here that is used in other tests (you are reusing two existing test accounts in this fix), it will be hard to detect if these specific emails because not only I have to check two additional accounts, I have to detect these specific emails among many other emails triggered by other tests.  

So the cons may not affect other devs, but they do make my life harder.